modules/gazelle/0.26.0/presubmit.yml

```diff
@@ -0,0 +1,25 @@
+matrix:
```

Comment:
This is not needed if you don't specify regular test & build targets for this module.

Reply:
Fixed.